repos/system_upgrade/el7toel8/actors/checkbind/libraries/model.py

```diff
@@ -1,19 +1,19 @@
 from leapp.models import BindFacts, BindConfigIssuesModel
-
 from leapp.libraries.common import isccfg
-from leapp.libraries.stdlib import api
-
 from leapp import reporting
 
+
 def add_statement(statement, state):
     """ Add searched statement to found issues """
+
     stmt_text = statement.serialize_skip(' ')
     name = statement.var(0).value()
     if name in state:
         state[name].append((stmt_text, statement.config.path))
     else:
         state[name] = [(stmt_text, statement.config.path)]
 
+
 def find_dnssec_lookaside(statement, state):
     try:
         assert statement.var(0).value() == 'dnssec-lookaside'
```

Comment:
just adding reasoning:
- it could introducing security risks and bugs into your applications
- syntax quirk that makes it easy to write useless assertions (assertions could be globally disabled by the interpreter)
https://dbader.org/blog/python-assert-tutorial

Reply:
okay, a bit dumb. It is good practice to leave assertions in place in C. Just disable them on compile time. I admit it would not work in python.